### PR TITLE
Migration: javax to jakarta

### PR DESCRIPTION
**Pull request by jbescos**
javax -> jakarta package rename #50

### CODE REVIEW

**Comment by m0mus** [comment]
@jbescos I approved, but forgot to mention that you should bump a major version number so it becomes `3.0.0-SNAPSHOT`

**Comment by jbescos** [comment]
I see, I have updated it.

File: api/src/main/java/jakarta/jws/soap/SOAPMessageHandler.java [hunk 1]
```diff
@@ -35,7 +35,7 @@
     /**
      * List of SOAP headers processed by the handler.  Each element in this array contains a QName which defines the
      * header element processed by the handler.  The QNames are specified using the string notation described in the
-     * documentation for javax.xml.namespace.QName.valueOf(String qNameAsString)
+     * documentation for jakarta.xml.namespace.QName.valueOf(String qNameAsString)
```
Comment (lukasj) [inline api/src/main/java/jakarta/jws/soap/SOAPMessageHandler.java]: this is not to be changed (part of JAXP/java.xml module on jdk 9+)

Comment (jbescos) [reply api/src/main/java/jakarta/jws/soap/SOAPMessageHandler.java]: It should be fine now

**Comment by jbescos** [comment]
> @jbescos I approved, but forgot to mention that you should bump a major version number so it becomes `3.0.0-SNAPSHOT`

Finally I reverted it because @lukasj told me there is a job that is taking care about this and it will conflict.

**Pull request merged**